apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts

```diff
@@ -0,0 +1,122 @@
+import { CustomerNoteFragment } from '../support/page_fragments/customer-note.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const customerNoteFragment = new CustomerNoteFragment();
+
+describe('Start picking a picklist with customer note', () => {
+  let identifier = '';
+
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.visit('/');
+    pickingListsFragment
+      .getCustomerNoteButton()
+      .parents('oryx-picking-list-item')
+      .within(() => {
+        pickingListsFragment
+          .getIdentifier(true)
+          .invoke('text')
+          .then((text) => {
+            identifier = text.trim();
+
+            pickingListsFragment.getStartPickingButton(true).click();
+          });
+      });
```

Comment:
It would be good to extract this into a method in `pickingListsFragment`, e.g. `pickingListsFragment.getIdentifier()`

Also, instead of using `let identifier;` we should use aliases - see https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Variables for details